### ISSUE TITLE
CSS .date float

### ISSUE THREAD

**Issue by mroswell** [DjangoGirls/tutorial#457]
In the CSS lesson 19, 
https://github.com/DjangoGirls/tutorial/blob/master/en/css/images/final.png
shows a left-aligned date.

But the CSS designated for blog.css at https://github.com/DjangoGirls/tutorial/tree/master/en/css calls for right float.

```
.date {
    float: right;
    color: #828282;
}
``` 
I suggest updating the screenshot accordingly (or updating the CSS).

**Comment by kojoidrissa** [comment]
I submitted a [pull request](https://github.com/DjangoGirls/tutorial/pull/456) that changed the CSS from `float: right;` to `float: none;` (which was probably bad style...should've just done left or removed the line), but then I saw in [Lesson 21](http://tutorial.djangogirls.org/en/extend_your_application/index.html), the screenshots showed the date as right-aligned.

So, I'd suggest a new screen shot for the CSS lesson.

I'll submit a 2nd pull request to amend my first pull request to change the CSS back to `float: right;` (I'd made one other small change). I may not be able to get around to doing another screenshot today. If someone else can do that, that'd be great.
